Makefile

```diff
@@ -81,30 +81,32 @@ checkgenerate:
 
 .PHONY: dockercomposetestgo
 dockercomposetestgo: dockercomposeclean
-	docker-compose run client-connect-to-server-connect-h1
-	docker-compose run client-connect-to-server-connect-h2
-	docker-compose run client-connect-to-server-connect-h3
-	docker-compose run client-connect-grpc-to-server-connect-h1
-	docker-compose run client-connect-grpc-to-server-connect-h2
-	docker-compose run client-connect-grpc-web-to-server-connect-h1
-	docker-compose run client-connect-grpc-web-to-server-connect-h2
-	docker-compose run client-connect-grpc-web-to-server-connect-h3
-	docker-compose run client-connect-grpc-web-to-envoy-server-connect-h1
-	docker-compose run client-connect-grpc-web-to-envoy-server-grpc-h1
-	docker-compose run client-connect-grpc-to-server-grpc
-	docker-compose run client-grpc-to-server-connect
-	docker-compose run client-grpc-to-server-grpc
+	docker-compose run client-connect-go-to-server-connect-go-h1
+	docker-compose run client-connect-go-to-server-connect-go-h2
+	docker-compose run client-connect-go-to-server-connect-go-h3
+	docker-compose run client-connect-go-to-server-connect-node-fastify-h1
+	docker-compose run client-connect-go-to-server-connect-node-fastify-h2
```

Comment:
We're running connect-go with the Connect protocol against the connect-es fastify server over HTTP/1.1 and HTTP/2, both cleartext. Nice!

Can we also run:
- connect-go with the gRPC protocol over H1
- connect-go with the gRPC protocol over H2
- connect-go with the gRPC-web protocol over H1
- connect-go with the gRPC-web protocol over H2
- grpc-go (always the gRPC protocol, always H2)

This will give us good coverage over most code paths in the connect-es server.